pkg/resourceprovider/card.go

```diff
@@ -51,13 +49,23 @@ func PostCard(id string, challange string, difficulty string) error {
 	cardcmd := exec.Command(tmpFile.Name(), id, challange, difficulty)
 	output1, err := cardcmd.Output()
 	if err != nil {
-		log.Debug().
-			Str("pow->event", "PowNewPowRound").
+		log.Error().
 			Msgf("execute temp file failed: %v", err)
 		return err
 	}
 
 	// Print the output
 	println(string(output1))
+
+	cardInfo := make(map[string]string)
```

Comment:
@arsen3d can i parse the output you the binary?

Reply:
i notice you make a change to post payload in the binary, so can i close this pr?

Reply:
I believe you have to close the channel too no?